bears/coffee_script/CoffeeLintBear.py

```diff
@@ -133,69 +133,68 @@ def generate_config(filename, file,
         :param force_braces:
             Prohibits implicit braces when declaring object literals.
 
-            Example: If ``force_braces = True`` then
-            ```
-            1:2, 3:4
-            ```
-            is prohibited, whereas
-            ```
-            {1:2, 3:4}
-            ```
+            Example: If ``force_braces = True`` then::
+
+                1:2, 3:4
+
+            is prohibited, whereas::
+
+                {1:2, 3:4}
+
             is accepted.
         :param allow_implicit_parentheses:
             Allows implicit parentheses.
         :param allow_interpolation_in_single_quotes:
             Allows string interpolation in a single quoted string.
 
-            Example: If ``allow_interpolation_in_single_quotes = False`` then
-            ```
-            f = '#{bar}'
-            ```
-            is prohibited, whereas
-            ```
-            f = "#{bar}"
-            ```
+            Example: If ``allow_interpolation_in_single_quotes = False`` then::
+
+                f = '#{bar}'
+
+            is prohibited, whereas::
+
+                f = "#{bar}"
+
             is correct.
         :param allow_stand_alone_at_sign:
             Allows the use of stand alone  ``@``.
 
-            Example: If ``allow_stand_alone_at_sign = False``
-            ```
-            @ notok
-            not(@).ok
-            @::
-            ```
-            are prohibited, whereas
-            ```
-            @alright
-            @(fn)
-            @ok()
-            @[ok]
-            @ok()
-            ```
-            are accepted.
```

Comment:
Thank you for carefully reviewing.  Could you do a follow up patch, re-adding it?  Be sure to mention this commit in your commit message, and use the same issue #.